src/pyodide.js

```diff
@@ -330,6 +331,37 @@ var languagePluginLoader = new Promise((resolve, reject) => {
   Module.preloadedWasm = {};
   let isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
 
+  Module.runPython = function(
+      code) { return Module.py_pyodide.eval_code(code, Module.globals); };
+
+  Module.loadPackagesForCode =
+      async function(code, messageCallback, errorCallback) {
+    let imports = Module.py_pyodide.find_imports(pycode);
+    if (imports.length) {
+      let packageNames =
+          self.pyodide._module.packages.import_name_to_package_name;
+      let packages = new Set();
+      // HC: What the heck does this do? I don't like the look of it.
+      for (let name of jsimports) {
+        if (name in packageNames) {
+          packages.add(name);
+        }
+      }
+      if (packages.size) {
+        await loadPackage(Array.from(packages.keys()), messageCallback,
+                          errorCallback);
+      }
+    }
+  }
```

Comment:
Putting a semicolon here would most likely significantly improve the formatting.

Reply:
We should really try to switch to a js linter for javascript code rather than using clang-format.